monkey/monkey_island/cc/services/agent_binary_service/flask_resources/agent_binaries_masque.py

```diff
@@ -0,0 +1,47 @@
+from http import HTTPStatus
+from json import JSONDecodeError
+
+from flask import make_response, request
+from flask_security import auth_token_required, roles_accepted
+
+from monkey_island.cc.flask_utils import AbstractResource
+from monkey_island.cc.services.authentication_service import AccountRole
+
+from .. import IAgentBinaryService
+
+
+class AgentBinariesMasque(AbstractResource):
+    urls = ["/api/agent-binaries/<string:os>/masque"]
+
+    def __init__(self, agent_binary_service: IAgentBinaryService):
+        self._agent_binary_service = agent_binary_service
+
+    @auth_token_required
+    @roles_accepted(AccountRole.AGENT.name, AccountRole.ISLAND_INTERFACE.name)
+    def get(self, os):
+        try:
+            get_os_masque = getattr(self._agent_binary_service, f"get_{os}_masque")
+
+            masque = get_os_masque()
+            return make_response({"masque": masque}, HTTPStatus.OK)
```

Comment:
This should be returning raw bytes/octets, not JSON. In fact, I don't believe JSON supports  bytes, so we're missing a unit test that tests with non-printable characters.